src/com/magento/idea/magento2plugin/actions/context/xml/NewAclXmlAction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+package com.magento.idea.magento2plugin.actions.context.xml;
+
+import com.intellij.ide.fileTemplates.actions.AttributesDefaults;
+import com.intellij.psi.PsiDirectory;
+import com.intellij.psi.PsiFile;
+import com.magento.idea.magento2plugin.actions.context.AbstractContextAction;
+import com.magento.idea.magento2plugin.magento.files.ModuleAclXml;
+import com.magento.idea.magento2plugin.magento.packages.ComponentType;
+import com.magento.idea.magento2plugin.magento.packages.Package;
+import com.magento.idea.magento2plugin.util.magento.GetMagentoModuleUtil;
+import org.jetbrains.annotations.NotNull;
+
+public class NewAclXmlAction extends AbstractContextAction {
+
+    public static final String ACTION_NAME = "Magento 2 ACL File";
+    public static final String ACTION_DESCRIPTION = "Create a new Magento 2 acl.xml file";
+
+    /**
+     * New routes.xml file generation action constructor.
```

Comment:
Please, change annotation to appropriate one.

Reply:
Hi @bohdan-harniuk .
Thanks for your code review. I fixed. Please check.